walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +123,61 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsidered2() {
```

Comment:
Prefer to name as an explicit test case. The naming pattern is 
testCase_behavior_expectedResult 

The format might be flexible, but the name should be descriptive.

For example, this test mighe be:
public void query_multipleAttendees_allScheduled() {}

Might be worthwhile putting some thinking to name the test case, especically to distinguish from other test cases.